tests/subsys/sd/mmc/boards/rcar_h3ulcb_ca57.overlay

```diff
@@ -0,0 +1,8 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+&emmc2 {
+	disk {
+		status = "okay";
+	};
+	status = "okay";
+};
```

Comment:
Why is it not enabled on the board level, and why is it worthy to pollute samples and tests with these specific overlays?

Reply:
> Because we don't need to enable everything that we support. Don't forget that this is a real-time OS, and some users may not want to use SD/MMC, for example. So, why should it be enabled by default?

So because of real-time, you say? okay, then do not enable the SD/MMC drivers by default, this is not usually done by node enabled status alone (but if it is, it is a bug).

Reply:
> > this is not usually done by node enabled status alone (but if it is, it is a bug).
> 
> If I understand you correctly, your point is opposite to this one: [#60285 (comment)](https://github.com/zephyrproject-rtos/zephyr/pull/60285#discussion_r1262717048)

Usually there is major Kconfig option like SERIAL or here SDHC. Kconfig options like SERIAL or SDHC disable all drivers, regardless of node status and derived driver specific Kconfig option status. If emmc2 node describes hardware on your board then it should be described on the board level. But Kconfig option SDHC should not be enabled per default, see  https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html#general-recommendations.

Reply:
What is being asked here is that the MMC device be added/enabled at the board level versus sample level. As @jfischer-no mentioned, unless `CONFIG_SDHC=y`, the SD host controller driver will not be enabled. You can see many example of this in Zephyr board devicetrees. For example, many boards enable SPI and I2C controllers, but those drivers won't be built unless `CONFIG_SPI` or `CONFIG_I2C` are turned on

Reply:
@xakep-amatop revisiting this- this PR has a *lot* of overlays, pretty much one per each test. Could you instead enable the common set of Kconfigs for these tests in the board level `Kconfig.defconfig`, like so?

```
if MMC_STACK

config MMC_RCA
	default 1

endif
```